requirements-dev.txt

```diff
@@ -1,6 +1,6 @@
 qiskit-aer>=0.13
 pytest
-pylint==2.16.2
```

Comment:
I worry this might be too loose for pylint which notoriously has had issues with rule stability between even on bugfix releases. But lets give it a try and we can do an exact version pin if needed